dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/TracerBuilder.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2019 Datadog, Inc.
+ */
+
+package com.datadog.android.tracing
+
+import datadog.opentracing.DDSpan
+import datadog.opentracing.DDTracer
+
+/**
+ * An interface enabling Datadog tracing features.
+ *
+ * It allows you to create a custom DDTracer which can be used to create and use a [DDSpan]
+ * for tracing your application workflow.
+ *
+ * You can have multiple loggers configured in your application, each with their own settings.
+ */
+interface TracerBuilder {
+    /**
+     * Builds a [DDTracer] based on the current state of this Builder.
+     */
+    fun build(): DDTracer
```

Comment:
WDYT about hiding `datadog.opentracing.DDTracer` types under our own umbrela? So exposing `Tracer` instead of `DDTracer`? Basic `typealias` could work initially, and if we have to differentiate our `Tracer` somehow (comparing to `DDTracer`), we'll be able to introduce own type wrapping `DDTracer`.

Reply:
You cannot do that. Users need DDTracer for using the Public API. DDTracer is exposed in the _dd-trace-ot_ API and should be exposed also on our end, otherwise you will have to duplicate and delegate all the methods in DDTracer inside your Tracer which doesn't make sense. Furthermore users are already used with the DDTracer and all the Public API exposed in the JAVA API.

Reply:
Done